server/routes/api.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const urllib = require('urllib');
+
+const teamToIDs = {
+  lakers: '1610612747',
+  warriors: '1610612744',
+  heat: '1610612748',
+  suns: '1610612756',
+};
+
+let nba = {
+  data: {},
+};
+let dreamTeam = [];
+
+urllib.request('http://data.nba.net/10s/prod/v1/2018/players.json', function (err, data) {
+  if (err) {
+    console.log(err);
+  }
+  nba.data = JSON.parse(data).league.standard;
+});
+
+router.get('/teams/:teamName', function (request, response) {
+  const teamId = teamToIDs[request.params.teamName];
+  const teamData = nba.data.filter((p) => p.teamId === teamId && p.isActive);
+  const filteredData = teamData.map((a) => ({
+    id: a.personId,
+    firstName: a.firstName,
+    lastName: a.lastName,
+    pos: a.pos,
+    jerseyNum: a.jersey,
+    dreamer:false,
+    imgUrl: `https://nba-players.herokuapp.com/players/${a.lastName}/${a.firstName}`,
+  }));
+  response.send(filteredData);
+});
+
+router.get('/player/:lastName/:firstName', function (req, res) {
+  const lastName = req.params.lastName;
+  const firstName = req.params.firstName;
+  urllib.request(
+    `https://nba-players.herokuapp.com/players-stats/${lastName}/${firstName}`,
+    function (err, data) {
+      if (err) {
+        throw err;
+      }
+      try {
+        data = JSON.parse(data);
+        filteredDataA = {
+          games_played: data.games_played,
```

Comment:
not sure, but i think you could do something like this:

games_played: data.games_played || 'N-A' (etc.)

instead of the error handling